documentation/NUnit1031.md

```diff
@@ -0,0 +1,101 @@
+# NUnit1031
+
+## The individual arguments provided by a ValuesAttribute must match the type of the corresponding parameter of the method
+
+| Topic    | Value
+| :--      | :--
+| Id       | NUnit1031
+| Severity | Error
+| Enabled  | True
+| Category | Structure
+| Code     | [ValuesUsageAnalyzer](https://github.com/nunit/nunit.analyzers/blob/master/src/nunit.analyzers/ValuesUsage/ValuesUsageAnalyzer.cs)
+
+## Description
+
+The individual arguments provided by a ValuesAttribute must match the type of the corresponding parameter of the method.
+
+## Motivation
+
+To prevent tests that will fail at runtime due to improper construction.
+
+## How to fix violations
+
```

Comment:
```suggestion
```

Reply:
I copied the style from NUnit1001.md (https://github.com/nunit/nunit.analyzers/blob/master/documentation/NUnit1001.md?plain=1).  I'll push changes in line with your suggestions, just fyi that we should probably update NUnit1001 (and/or others) when we have time.

Reply:
Thanks, I reverted the changes so the two align.